cleanlab/count.py

```diff
@@ -1415,6 +1415,6 @@ def _get_confident_thresholds_multilabel(
     for class_num, (label, pred_prob) in enumerate(zip(y_one.T, pred_probs.T)):
         pred_probabilitites = stack_complement(pred_prob)
         confident_thresholds[class_num] = get_confident_thresholds(
-            pred_probs=pred_probabilitites, labels=labels
+            pred_probs=pred_probabilitites, labels=label
         )
```

Comment:
```suggestion
            pred_probs=pred_probs_binary, labels=label_for_class
        )
```

Reply:
The bug appeared because of not using precise enough variable names. We want to be as precise as possible in the variable names for code clarity and avoiding bugs like this (and should never have typos in variable names like `pred_probabilitites`)